pydrive2/files.py

```diff
@@ -518,6 +518,36 @@ def Delete(self, param=None):
         """
         self._FilesDelete(param=param)
 
+    @LoadAuth
+    def Copy(self, target_folder, new_title = None, param=None):
+        """Copy this file to a new location.
+
+        :param target_folder: Folder where the file will be copied.
+        :type target_folder: GoogleDriveFile.
+        :param new_title: Name of the new file.
+        :type new_title: str.
+        :param param: addition parameters to pass
+        :type param: dict
+        """
+
+        if new_title is None:
+            new_title = self["title"]
```

Comment:
Question: is it required? I mean, do we have to provide the title? 

I'm asking because this `self["title"]` call might require hitting an API to fetch the metadata (which is an expensive operation).

Reply:
if you don't provide a title, the filename of the copied file will be "Copy of [old-filename]".
So, yes, we can remove those two lines, if the user want to use the same title it will have to pass it.